src/routes/verify-email-address/+page.svelte

```diff
@@ -0,0 +1,80 @@
+<script lang="ts">
+	import * as Menu from 'menu/verify';
+	import * as MenuItem from 'menu/menuItems';
+
+	import StyledLink from 'elements/input/styledLink.svelte';
+	import Footer from 'elements/navigation/footer.svelte';
+	import Header from 'elements/navigation/header.svelte';
+	import HeadlinePage from 'elements/text/headlinePage.svelte';
+	import TextLine from 'elements/text/textLine.svelte';
+
+	export let data;
+</script>
+
+<div class="verify-email-page-wrapper">
+	<Header menu={Menu.header} />
+	<div class="verify-email-content-wrapper">
+		<HeadlinePage>E-Mail Verifizierung</HeadlinePage>
+		{#if data.success === undefined}
+			<div class="verify-email-text-wrapper">
+				<TextLine>👻 Wir versuchen gerade deine E-Mail zu verifizieren. 👻</TextLine>
+				<TextLine>Offenbar dauert das alles ein bisschen länger als wir erwartet haben.</TextLine>
+				<TextLine>Bitte hab etwas Geduld</TextLine>
+			</div>
+			<TextLine>Falls sich hier länger nix tun sollte melde dich gerne bei uns.</TextLine>
+			<StyledLink
+				text={MenuItem.imprint.name}
+				href={MenuItem.imprint.url}
+				title={MenuItem.imprint.description}
+				newTab={false}
+			/>
+		{:else if data.success}
+			<div class="verify-email-text-wrapper">
+				<TextLine>Deine E-Mail wurde erfolgreich verifiziert.</TextLine>
+				<TextLine>Du kannst dich ab jetzt anmelden.</TextLine>
```

Comment:
Just personal preference...I like this a bit more:
```suggestion
				<TextLine>Du kannst dich jetzt anmelden.</TextLine>
```